packages/astro/src/@types/astro.ts

```diff
@@ -2390,15 +2392,33 @@ interface AstroSharedContext<
 	currentLocale: string | undefined;
 }
 
+/**
+ * The `APIContext` is the object made available to endpoints and middleware.
+ * It is a subset of the `Astro` global object available in pages.
```

Comment:
I thought they were the same. What are the differences?

Reply:
The difference is that the Astro global also has `.slots` and `.glob`.